content/docs/studio/user-guide/model-registry/download-models.md

```diff
@@ -1,28 +1,75 @@
-# Download models
+# Use models
+
+Whether you need to download your models to use them, or you're looking to set
+up some automation in CI/CD to deploy them, Iterative Studio provides these
+capabilities.
+
+## Download models
 
 If your model file is DVC-tracked, you can download any of its registered
-versions using [dvc get](https://dvc.org/doc/command-reference/get).
+versions using `dvc get`.
+
+### Looking up right command in Studio
+
+Model details page have a command ready to copy and use under "Get the model
+file": ![](https://hackmd.io/_uploads/SkE5qUGr2.png)
+
+### Constructing the command manually
+
+You can also construct this command manually using this template:
 
 ```cli
 $ dvc get ${GIT_REPO} ${MODEL_PATH} --rev ${GIT_REV}
 ```
 
-For example, one of the models in the
-[bank customer churn demo project](https://github.com/iterative/demo-bank-customer-churn)
-is `lightgbm-model`. You can find its registered versions in the
-[list of Git tags](https://github.com/iterative/demo-bank-customer-churn/tags).
+Let's download the model in the
+[example-get-started-experiments](https://github.com/iterative/example-get-started-experiments)
+is `seg-model`.
 
-Suppose you want to dowload the model file for the `lightgbm-model@v2.0.1`
-version. In the model details page, you can see that the model file path is
-`.mlem/model/lightgbm-model`.
+To find out the model file path (e.g. `models/model.pkl`) you can see the
+Model's details page or check `dvc.yaml` the model annotation was written to.
 
-To download the file, you can use the following dvc command:
+To find out the Git revision, you can check the
+[list of Git tags](https://github.com/iterative/example-get-started-experiments/tags)
+or use [GTO](https://mlem.ai/doc/gto/command-reference/show/):
```

Comment:
Can you find the rev in the model details page?

Reply:
Not yet. I'll add an issue to Studio.

Reply:
https://github.com/iterative/studio/issues/6014